lbrynet/wallet/transaction.py

```diff
@@ -72,6 +72,15 @@ class Transaction(BaseTransaction):
     input_class = Input
     output_class = Output
 
+    @classmethod
+    def pay(cls, amount: int, address: bytes, funding_accounts: List[Account], change_account: Account):
+        ledger = cls.ensure_all_have_same_ledger(funding_accounts, change_account)
+        hash_of_address = ledger.address_to_hash160(address)
+
+        output = Output.pay_pubkey_hash(amount, hash_of_address)
+
+        return cls.create([], [output], funding_accounts, change_account)
```

Comment:
Can you reduce the number of blank lines and inline `hash_of_address` to be consistent with how all of the other `Transaction` methods are implemented.